### DISCUSSION TITLE
All new tab icons are black and white instead of their original colors.

### DISCUSSION THREAD

**Discussion by PhilipPhil** [microsoft/MicrosoftEdge-Extensions#94] [Q&A]
When creating a new tab app, the icon on the top of the tab will always be black and white instead of the original colors. For example, if a circle png is added to a globe the only thing visible will be a black ball. The app works fine on Chrome and other Chromium browsers I've tried. Only Edge has this issue. Is this a feature or a bug? I can't find any information about this. Sample picture below on the left we have the new tab tab icon which is not the correct color, and on the right we have the correct icon and what should be shown.

![Screenshot 2023-06-23 182731](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/44811794/f7b13e42-6df6-4f4c-ae75-28f400426a53)

**Comment by Gurjeet-msft** [comment]
@PhilipPhil thank you for reaching out to us. 

Apologies for the inconvenience caused. 

Could you please confirm if the above issue occurred while using any extension on Microsoft Edge or just occurred on when you were using the Microsoft Edge irrespective of any browser extension.

**Comment by PhilipPhil** [comment]
This issue occurs on every new tab extension on edge.

**Comment by Gurjeet-msft** [comment]
@PhilipPhil` Thanks for confirming, this issue looks like a Browser specific.

Could you please reach out to our **Microsoft Edge** Dev team, they should be able to help you with this. 

**Github:**  https://github.com/MicrosoftEdge
**Twitter:**  https://twitter.com/msedgedev

Thanks.

**Comment by skyroor** [comment]
Was there a solution to this problem ever found?

**Comment by ManikanthMSFT** [comment]
Hey @skyroor, could you please provide the details of the extension you are facing issues with? This will help us review and assist you further.

**Reply by skyroor** [reply]
Same issue as original issue. I am developing a new tab page extension and any icon/PNG/icon used in the manifest will be masked out and only show as white for the favicon icon.

Again, this is only an issue in Edge. Firefox and Chrome show the proper unmasked and colored icon. All other icons in edge from the extension are fine (extension settings page, toolbar, etc).

My theory is that edge is apply some intentional filtering for their icon that is carried over to all favicon for new tab pages. I don't know if this is intentional, or a bit, or what it is. I would like to use my own colored icon though.

I have tested other new tab extensions and only one of them seemed to have a colored icon, and I'm going to test why that might be later today. Might be linked to redirecting the page html page, and not setting it as the default new tab page in the manifest.

**Comment by PhilipPhil** [comment]
No solution was found. You can pick up any new tab extension and see this
issue. The icon will be black and white instead of the original one.

**Reply by skyroor** [reply]
I posted a reply above what my theory is, will be testing it later to confirm. Really annoying issue though. However, thanks for updating me!

**Comment by PhilipPhil** [comment]
I spoke to support. They said it is a feature for some reason.

**Reply by skyroor** [reply]
Ahh, got it! Well, forced aesthetics on custom extensions is a nice "feature".

**Reply by prem-k-r** [reply]
> I spoke to support. They said it is a feature for some reason.

Ah, a bug disguised as a feature!

**Comment by apkqad** [comment]
how can I access the edge theme store I need to change the theme of my edge [Homepage](https://apkqad.com/), is there any mod file or something else available???
